src/Youtube.js

```diff
@@ -298,26 +298,28 @@ THE SOFTWARE. */
 
     onPlayerError: function(e) {
       this.errorNumber = e.data;
+      this.trigger('pause');
       this.trigger('error');
-
-      this.ytPlayer.stopVideo();
     },
 
     error: function() {
       switch (this.errorNumber) {
```

Comment:
unfortunately, we can't just use `errorNumber` here because the codes have specific meaning in videojs and the HTML: https://github.com/videojs/video.js/blob/master/src/js/media-error.js#L106-L112.
In videojs-errors, we opted for using negative code numbers for "custom" errors: https://github.com/brightcove/videojs-errors/blob/master/src/plugin.js#L8-L49.
We should see if we can align with the native error codes or the ones from videojs-errors, otherwise, we should choose non-conflicting error codes.

For example, errorNumber 2 and 100 could potentially be equivalent to error code 4.
video.js uses the message provided over the default message. We could then add youtube's errorNumber to the message.